test/table.ts

```diff
@@ -1262,7 +1262,9 @@ describe('Bigtable/Table', () => {
           // eslint-disable-next-line @typescript-eslint/no-explicit-any
           ((stream: any) => {
             stream.push([{key: 'a'}]);
-            stream.emit('error', makeRetryableError());
+            setImmediate(() => {
+              stream.emit('error', makeRetryableError());
+            });
```

Comment:
please extract to a helper function and add a comment why this is necessary

Reply:
Done